spec.md

```diff
@@ -1957,6 +2319,72 @@ asClock(x)
 [@sec:primitive-operations] will describe the format and semantics of each
 primitive operation.
 
+## Reading Probe References
+
+Probes are read using the `read`{.firrtl} operation.
+
+Read expressions have source flow and can be connected to other components:
+
+```firrtl
+module Foo :
+  output p : Probe<UInt>
+  ; ...
+
+module Bar :
+  output x : UInt
+
+  inst f of Foo
+  x <= read(f.p) ; indirectly access the probed data
+```
+
+Indexing statically (sub-field, sub-index) into a probed value is allowed as
+part of the read:
+
+```firrtl
+module Foo :
+  output p : Probe<{a: UInt, b: UInt}>
+  ; ...
+
+module Bar :
+  output x : UInt
+
+  inst f of Foo
+  x <= read(f.p.b) ; indirectly access the probed data
```

Comment:
is it explicitly not legal to do:

```
x <= read(f.p).b ; indirectly access the probed data
```

Reply:
I would be careful not to confuse internal IR structure with FIRRTL structure.  firrtl isn't a one-op per line language, so I would expect read(f.p).b to work.

Reply:
Agreed!  Fixed.